### PR TITLE
P14C-6392: added ssl configuration

### PR DESCRIPTION
**Pull request by Deivur**
Added SSL configuration.

### CODE REVIEW

File: spring-boot/pom.xml [hunk 1]
```diff
@@ -5,16 +5,16 @@
...
   <groupId>com.pingidentity.samples</groupId>
   <artifactId>user-registration-sample</artifactId>
-  <version>0.0.4-SNAPSHOT</version>
+  <version>0.0.5-SNAPSHOT</version>
 
   <parent>
     <groupId>org.springframework.boot</groupId>
     <artifactId>spring-boot-starter-parent</artifactId>
-    <version>2.1.6.RELEASE</version>
+    <version>2.2.4.RELEASE</version>
```
Comment (hkrop) [inline spring-boot/pom.xml]: If this version works properly, then u can use `spring-boot.version` property here

File: spring-boot/src/main/resources/application.yml [hunk 2]
```diff
@@ -1,5 +1,10 @@
 server:
-  port: 8080
+  port: 8443
+  ssl:
+    key-store-type: <key store type>
```
Comment (hkrop) [inline spring-boot/src/main/resources/application.yml]: Please specify in README wht this attributes are and how to generate them

File: spring-boot/pom.xml (hunk 1 above)
Comment (Deivur) [reply spring-boot/pom.xml]: Nope, it doesn't work. Just removed `spring-boot.version` property as it is not used anywhere.

File: spring-boot/src/main/resources/application.yml (hunk 2 above)
Comment (Deivur) [reply spring-boot/src/main/resources/application.yml]: I did update for the Readme file.

File: spring-boot/pom.xml (hunk 1 above)
Comment (hkrop) [reply spring-boot/pom.xml]: I meant we have a property `spring-boot.version` that u can use here

File: spring-boot/README.md [hunk 3]
```diff
@@ -66,6 +66,12 @@ Most of PingOne platform scopes are self-explanatory, but if you need more detai
     - __Single Page or Web Application__ configuration in `spring.security.oauth2.client` path 
       - `<authorization_code_client_id>` with your client id (in `clientId` variable)
       - `<authorization_code_client_client_credentials_client_secret>` with your client secret (in `clientSecret` variable)
+    - __Security__:
+      - `<key store type>` the format used for the keystore.
+      - `<path to key>` the path to the keystore containing the certificate
+      - `<key store password>` the password used to generate the certificate
+      - `<key alias>` specifies the alias that you gave to the self-signed certificate when you created it
+<br>_`TIP:`_ ["how to create self-signed SSL certificate"](https://oracle-base.com/articles/linux/create-self-signed-ssl-certificates)
```
Comment (hkrop) [inline spring-boot/README.md]: Please adjust to
 

`<key store type>` the format used for the keystore (i.e PKCS12, JKS file)
` <key alias>` the alias mapped to self-signed certificate

Comment (Deivur) [reply spring-boot/README.md]: Done.

**Pull request merged**